src/App.css

```diff
@@ -40,9 +40,319 @@ a:hover {
     text-decoration: 3px underline #5bb6ea;
 }
 
+/* NavigationBar styles */
+.navigation-container {
```

Comment:
Since we created a separate file for the navigation bar (`src/ui/NavigationBar.tsx`), should we make a separate CSS file as well (e.g., `ui/NavigationBar.css`)?